src/components/Header/Notifications/GetNotifications.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable import/named */
```

Comment:
I think this file should not be here in the Header component folder, it could be better if you put it in the notification folder

Reply:
it is there because we need to get a number of notifications from the header